### PR TITLE
Genetic diversity text

### PR DESCRIPTION
**Pull request by cclarkson**
Results complete except section on IBD from @hardingnj.
Discussion paragraph in note form awaiting IBD results.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
 \subsection*{Genetic diversity within populations}
 %%
 
+%%
+We carried out investigations into the genetic diversity within populations to enable inference of their evolutionary histories.
```
Comment (hardingnj) [inline main.tex]: investigations into > analyses of?

File: main.tex [hunk 2]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+%
+An understanding of historical evolutionary trajectories is essential to parameterise mathematical models and predict future outcomes of vector control methodologies.
```
Comment (hardingnj) [inline main.tex]: methodologies >methods ?

File: main.tex [hunk 3]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+%@@ GET THE ACTUAL NUMBERS?
+All but three populations carry similar levels of median genetic diversity (from Guinea \textit{coluzzii} $\pi$ = 0.014 to Guinea \textit{gambiae} $\pi$ = 0.016), with only Angola ($\pi$ = 0.012), Mayotte ($\pi$ = 0.010) and Kenya ($\pi$ = 0.009) presenting lower levels (Figure \ref{div}a).
+%
+Mayotte and Kenyan samples also stood out in estimations of Tajima's D (which compares the average number of pairwise differences with the number of segregating sites), as the only populations carrying positive median values  (Mayotte Tajima's D = 1.11, Kenya Tajima's D = 1.97 Figure \ref{div}b).
+%
+The other populations presenting Tajima's D values between 0 and -2 (from Angola \textit{coluzzii} D = -0.24 to Cameroon \textit{gambiae} D = -2.058).
+%%
+
+%%
+The homozygosity of individuals can be informative with regard to population demography.
```
Comment (hardingnj) [inline main.tex]: ? Runs of homozygosity present within individuals

File: main.tex [hunk 4]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+%
+The additional colony cross parents in Phase 2 reveal a much broader spread in both count and fraction ROH between colonies, suggestive of different histories.
+%
+Ghana colony samples have some of lowest ROH counts and genome fraction found, even compared to wild caught samples, whereas some Kisumu colony samples have the highest counts and genomic fraction of ROH.     
+%
+ROH values from the Phase 2 novel island population of Mayotte were also striking, slightly elevated ROH fractions alongside the highest ROH counts found in natural populations.
+%%
+
+%%
+Identity by descent (IBD) metrics... (Figure \ref{div}d)@@Nick?
```
Comment (hardingnj) [inline main.tex]: Happy to do some work on this- I'll check the issue tracker

File: main.tex [hunk 5]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+%%
+We carried out investigations into the genetic diversity within populations to enable inference of their evolutionary histories.
+%
+An understanding of historical evolutionary trajectories is essential to parameterise mathematical models and predict future outcomes of vector control methodologies.
+%
+%@@ GET THE ACTUAL NUMBERS?
+All but three populations carry similar levels of median genetic diversity (from Guinea \textit{coluzzii} $\pi$ = 0.014 to Guinea \textit{gambiae} $\pi$ = 0.016), with only Angola ($\pi$ = 0.012), Mayotte ($\pi$ = 0.010) and Kenya ($\pi$ = 0.009) presenting lower levels (Figure \ref{div}a).
+%
+Mayotte and Kenyan samples also stood out in estimations of Tajima's D (which compares the average number of pairwise differences with the number of segregating sites), as the only populations carrying positive median values  (Mayotte Tajima's D = 1.11, Kenya Tajima's D = 1.97 Figure \ref{div}b).
+%
```
Comment (EricRLucas) [inline main.tex]: I would take out the first pair of brackets above and replace with a comma after "Tajima's D"

File: main.tex [hunk 6]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+The other populations presenting Tajima's D values between 0 and -2 (from Angola \textit{coluzzii} D = -0.24 to Cameroon \textit{gambiae} D = -2.058).
```
Comment (EricRLucas) [inline main.tex]: presenting -> presented?

File: main.tex [hunk 7]
```diff
@@ -241,6 +244,40 @@ \subsection*{Population structure}
...
+%%
+
+%%
+The homozygosity of individuals can be informative with regard to population demography.
+%
+Both counts and fraction of the genome carrying contiguous runs of homozygosity (ROH) can be indicative of inbreeding.
+%
+A previous analysis of Phase 1 populations \cite{Ag1000gConsortium2017}, saw Kenya stand out with ROH tracts taking up to ~60\% of the genome, a much higher percentage than all other natural populations and similar to the Mali lab colony samples (Figure \ref{div}c). 
+%
+The additional colony cross parents in Phase 2 reveal a much broader spread in both count and fraction ROH between colonies, suggestive of different histories.
```
Comment (EricRLucas) [inline main.tex]: This is a bit out of the blue talking about colony crosses. Will the reader already know what the crosses are and where the parents came from by this point? Is the message here that the different crosses have very different ancestry and so we can expect that they are independent data points?

**Comment by EricRLucas** [comment]
Looks great, thanks. Left some minor comments in the diff of the .tex file (not sure where they show up).

File: main.tex (hunk 1 above)
Comment (cclarkson) [reply main.tex]: Fixed.

File: main.tex (hunk 2 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 3 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 4 above)
Comment (cclarkson) [reply main.tex]: cool - will add issue.

File: main.tex (hunk 5 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 6 above)
Comment (cclarkson) [reply main.tex]: fixed.

File: main.tex (hunk 7 above)
Comment (cclarkson) [reply main.tex]: Ah yes, the crosses bit hasn't been added to the pop sampling results section yet, I will create an issue.

**Pull request merged**